client/users.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+    User Class for NodeBB Client
+"""
+from __future__ import unicode_literals
+
+from openedx.features.openedx_nodebb_discussion.client import Client
+
+from .utils import save_user_relation_into_db, get_nodebb_uid_from_username
+
+
+class NodeBBUser(Client):
```

Comment:
Why do we need to inherit the Client and create a `NodeBBUser` class?

Reply:
We are considering `client` as  a base class for all the required `nodebb classes` which will interact with `nodebb write api` so that those classes can interact with `nodebb write api` using the functionality of `client` class